### ISSUE TITLE
Pulmonary thromboembolism

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7380]
**Preferred term label:**
Pulmonary thromboembolism

**Synonyms**

**Definition (free text, please give PubMed ID)**
 Pulmonary embolism is caused by emboli, which have originated from venous thrombi, travelling to and occluding the arteries of the lung. 

 Pulmonary embolism (PE)  is the most dangerous form of venous thromboembolism, and undiagnosed or untreated PE can be fatal. Acute PE is associated with right ventricular dysfunction, which can lead to arrhythmia, haemodynamic collapse and shock. Furthermore, individuals who survive PE can develop post-PE syndrome, which is characterized by chronic thrombotic remains in the pulmonary arteries, persistent right ventricular dysfunction, decreased quality of life and/or chronic functional limitations. 
PMID: 29770793

**Parent term (use hpo.jax.org/app)**
Thromboembolism HP:0001907   2/19
OMIM:613426 

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
   

 PMID:18506004 

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
